gateway/src/apicast/metrics/3scale_backend_status.lua

```diff
@@ -0,0 +1,23 @@
+local prometheus = require('apicast.prometheus')
+local metrics_updater = require('apicast.metrics.updater')
+
+local _M = {}
+
+local backend_response_metric = prometheus(
+  'counter',
+  'backend_response',
+  "Response status codes from 3scale's backend",
+  { 'status' }
+)
+
+function _M.inc(status)
+  if status >= 200 and status < 300 then
```

Comment:
Wouldn't be better to use math ? 

```lua
string.format('%dxx', (404/100))
```

And handle `nil` and `0` separately.

Reply:
:+1: